v4/source/system.yaml

```diff
@@ -230,6 +230,40 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+    post:
+      tags:
+        - system
+      summary: Post logs
+      description: |
+        Post logs to the server logs.
+        ##### Permissions
+        Must have `manage_system` permission.
```

Comment:
And here maybe mention that if `ServiceSettings.EnableDeveloper` is set to true in the config file, then any user can use this endpoint